blocks/rich-text/editable.js

```diff
@@ -12,7 +12,7 @@ class Editable extends RichText {
 	constructor() {
 		super( ...arguments );
 		deprecated( 'Editable', {
-			version: '2.5',
+			version: '2.6',
```

Comment:
Should we change it in `deprecated.md` as well?

Reply:
>Should we change it in `deprecated.md` as well?

Yep. Added.